features/step_definitions/laundry_room_steps.rb

```diff
@@ -15,8 +15,8 @@
 end
 
 
-Given(/^I click on "([^"]*)" for first "([^"]*)"$/) do |element, data|
-  scope = find("tr[data-date='#{data}']")
+Given(/^I click on "([^"]*)" for first "([^"]*)"$/) do |element|
+  scope = find_link(class: ['booking-button'], :visible => :all).visible?
```

Comment:
If you end with `.visible?` you will get a `true` or `false` back. That cannot be used as a scope. Also, you probably need a better way to identify the link. Perhaps give it an id depending on that some dynamic value?